NumLib/DOF/DOFTableUtil.cpp

```diff
@@ -22,6 +22,8 @@ double getNodalValue(GlobalVector const& x, MeshLib::Mesh const& mesh,
 
     auto const index = dof_table.getLocalIndex(
         l, global_component_id, x.getRangeBegin(), x.getRangeEnd());
+    if (index == NumLib::MeshComponentMap::nop)
+        return 0.0;
```

Comment:
I'd think it is better to return something more harmful like NaN. But it would better to prevent such case at all with assert, which is probably more difficult.
Let's discuss this tomorrow.